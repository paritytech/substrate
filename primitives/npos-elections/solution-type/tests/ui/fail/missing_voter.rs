use sp_npos_elections_solution_type::generate_solution_type;

generate_solution_type!(pub struct TestSolution::<
	u16,
	TargetIndex = u8,
	Accuracy = Perbill,
>(8));

fn main() {}
