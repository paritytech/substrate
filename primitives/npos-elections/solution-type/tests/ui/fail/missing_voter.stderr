error: Expected binding: `VoterIndex = ...`
 --> $DIR/missing_voter.rs:4:2
  |
4 |     u16,
  |     ^^^
